Backend/src/main/java/com/project/omega/controller/OrderController.java

```diff
@@ -135,7 +135,7 @@ private void validateProductsExistence(List<OrderProductDto> orderProducts) thro
                 list.add(op);
             }
         }
-        if (!CollectionUtils.isEmpty(list)) {
+        if (CollectionUtils.isEmpty(list)) {
             new ProductNotFoundException("Product not found");

```

Comment:
Would this return a status 200 with an empty list?

Reply:
This is a function that is called in a put request, so it makes sense if any of the product ids fails to match to be a bad request.

It simply ensures that all bound products actually exist on the DB.